Shared/Managers/KeyboardManager.swift

```diff
@@ -34,34 +37,36 @@ class KeyboardManager {
         case none // No event has happe
     }
 
-    init() {
-        self.initialize()
-    }
-
     deinit {
         self.cancellables.forEach { cancellable in
             cancellable.cancel()
         }
     }
 
-    private func initialize() {
-        self.addKeyboardObservers()
-    }
+    func addKeyboardObservers(with inputAccessoryView: UIView? = nil) {
```

Comment:
This fixes an issue where the inputaccessory view may not get set before the updates come in. This ensures it does.